Update npm syslog docs

Mesverrum opened the pull request:
Summary: "There is a new guided install for this feature, so I added that as the default instructions and added a screen shot of the wizard."

If you got tagged to review this, it was a mistake and you can disregard!

zuluecho9 commented:
@Mesverrum Hi Marc. Could you give a brief summary of the work you were doing with this (e.g., what problem you were trying to solve)? Just helps us have to have a succinct summary when reviewing and if we potentially need to understand it better later.

Mesverrum commented:
Sorry,  had a lot of tabs open and forgot to fill in the description. There is a new guided install for this feature,  so I added that as the default instructions and added a screen shot of the wizard.

zuluecho9 commented:
I lowercases 'network performance monitoring' based on marketing guidelines (let me know if anyone has any questions and I'll point you to the resource that confirms that).
